.goreleaser.yml

```diff
@@ -3,6 +3,7 @@ release:
   github:
     owner: mattfenwick
     name: kubectl-cyclonus
+  prerelease: auto
```

Comment:
ahhh nice 👍 wasn't aware of this before

just looked at https://goreleaser.com/customization/release/ , my understanding of this field is that it `auto` is a better setting than the default of `false` -- is that accurate ?

thanks for fixing this !